src/macaron/__main__.py

```diff
@@ -217,6 +197,15 @@ def main() -> None:
         ),
     )
 
+    single_analyze_parser.add_argument(
+        "-pe",
+        "--provenance-expectation",
+        required=False,
+        default=[],
+        help=("The path to provenance expectation file or directory."),
+        action="append",
```

Comment:
Note: without providing `nargs="*"`, the user couldn't do something like:
```
macaron ... --provenance-expectation boo foo bah
```
Would this behavior be as expected ?

Reply:
See commit [649081b](https://github.com/oracle-samples/macaron/pull/192/commits/649081b04e1df8698e544df6bd65fefd5be995fd).

Reply:
I have decided disallowing a list of file paths altogether as it would complicate the `run_macaron.sh` script arg parsing. Instead users can put all the expectations in a directory.

See commit [6a71cf8](https://github.com/oracle-samples/macaron/pull/192/commits/6a71cf8de74d95541daa6ada412d8f6e2f64661d).